src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -205,7 +205,7 @@ const GeneralInfoStep = ({
           <AppTextArea
             fullWidth
             label={t('becomeTutor.generalInfo.textFieldLabel')}
-            maxLength={70}
+            maxLength={200}
```

Comment:
Change length in validations too: 
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/63456632/49025875-7a81-4e72-b36e-026b5614b973)

Reply:
You can see an error message when you type more than 70 letters.
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/83120263/871b9f80-e535-4055-9ece-3af1be0e37bd)

Reply:
Maybe we should remove this validation as it will not work out for the same length as on import